### DISCUSSION TITLE
Changing terminal size is not persistent with a server restart. Config always resets to default values.

### DISCUSSION THREAD

**Discussion by IOcase** [cc-tweaked/CC-Tweaked#1986] [Q&A] [answered]
I'm trying to change my terminal size for both the terminal and the pocket computer. I've tried changing the numbers by extremely small values and it still defaults back to the original numbers in ../serverconfig/computercraft-server.toml.

Any idea why this is happening? I figured maybe it was pulling from my /config folder to apply settings globally so I copied the .toml into there but no change... I've tried searching and I can't find much except to change the values where I'm already changing the values... ChatGPT is less than helpful also. Thanks in advance.

**Comment by SquidDev** [comment, marked as answer]
I'm assuming you're using Forge 1.20.1? There's several issues with Forge's config system, that can cause it to overwrite files when the server shuts down.

If you've not tried already, can you try shutting down the server, editing the config file, and then starting the server again?

**Comment by IOcase** [comment]
That worked, thanks!
